test/lit/exec/fuzzing-api.wast

```diff
@@ -118,6 +120,139 @@
    )
   )
  )
+
+ ;; CHECK:      [fuzz-exec] calling ref.calling
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 42]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 3.14159]
+ ;; CHECK-NEXT: [exception thrown: __private ()]
```

Comment:
Where does this `__private ()` come from? It doesn't look like a normal trap, but shouldn't it be a normal trap?

Reply:
It is an exception because it is the same behavior as JS doing a call of a null, which throws. We do this for all the places JS would throw, in `execution-results.h` of these "ask JS to do something" APIs.